### PR TITLE
Fix 3859 Change map view action does not work in openlayer

### PR DESCRIPTION
**Pull request by allyoucanmap**
## Description
This PR allows default projections to trigger CHANGE_MAP_VIEW even if the center is outside the max extension of the projections. (EPSG:3857, EPSG:900913 and EPSG:4326) 

## Issues
 - #3859

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)

 - [x] Bugfix

**What is the current behavior?** (You can also link to an open issue here)
#3859

**What is the new behavior?**
This PR allows default projections to trigger CHANGE_MAP_VIEW even if the center is outside the max extension of the projections. (EPSG:3857, EPSG:900913 and EPSG:4326) 

**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: web/client/components/map/openlayers/Map.jsx [hunk 1]
```diff
@@ -412,9 +412,14 @@ class OpenlayersMap extends React.Component {
         let view = this.map.getView();
         let tempCenter = view.getCenter();
         let projectionExtent = view.getProjection().getExtent();
+        const crs = view.getProjection().getCode();
+        // some projections are reapeated on the y axis
```
Comment (MV88) [inline web/client/components/map/openlayers/Map.jsx]: *repeated

File: web/client/components/map/openlayers/__tests__/Map-test.jsx [hunk 2]
```diff
@@ -891,4 +891,120 @@ describe('OpenlayersMap', () => {
...
     });
+
+    it('test updateMapInfoState with custom projection 3003 (not listed in wrappedProjections)', () => {
+
+        const CENTER_OUTSIDE_OF_MAX_EXTENT = {
+            x: 50,
+            y: 70,
+            crs: 'EPSG:4326'
+        };
+        const testHandlers = {
```
Comment (MV88) [inline web/client/components/map/openlayers/__tests__/Map-test.jsx]: I would change the handler name and related spy to a more specific one, maybe:
spy  ==> spyOnMapViewChanges
handler ==> onMapViewChanges

if you prefer to use a more general names is fine too.

**Comment by MV88** [comment]
@allyoucanmap just a comment on tests, feel free to merge this as is

File: web/client/components/map/openlayers/Map.jsx [hunk 3]
```diff
@@ -412,9 +412,14 @@ class OpenlayersMap extends React.Component {
         let view = this.map.getView();
         let tempCenter = view.getCenter();
         let projectionExtent = view.getProjection().getExtent();
+        const crs = view.getProjection().getCode();
+        // some projections are repeated on the y axis
```
Comment (offtherailz) [inline web/client/components/map/openlayers/Map.jsx]:
```suggestion
        // some projections are repeated on the x axis
```

**Pull request merged**